src/Core/src/Handlers/Image/ImageHandler.cs

```diff
@@ -21,6 +21,10 @@ public partial class ImageHandler : IImageHandler
 #if __ANDROID__ || WINDOWS || TIZEN
 			[nameof(IImage.Background)] = MapBackground,
 #endif
+			[nameof(IImage.Height)] = MapHeight,
+			[nameof(IImage.Width)] = MapWidth,
+			[nameof(IImage.MaximumHeight)] = MapMaximumHeight,
```

Comment:
Does this replace the ViewHandler mapping of MaximumHeight and MaximumWidth? Do these values (and Height and Width) work correctly when the Aspect is not set to AspectFill and there is not Background?

Reply:
How do I check if it replaces the ViewHandler mappings? It's working as expected with the other aspects.

Reply:
It looks like this in my branch and in main: 
![image](https://github.com/dotnet/maui/assets/59936622/b5d967dc-2780-4ab5-ac9c-c91a59fecbfa)

Reply:
Take the `BackgroundColor` property off of the Image. (Set it on the ContentPage if you need to, or set your device to light mode, but leave it off of the Image.)